plugins/lang/admincmd.txt

```diff
@@ -226,79 +226,79 @@ ADMIN_EXTEND_2 = ADMIN %s: produziti mapu za %d minuta
 MAP_EXTENDED = Mapa "%s" je produzena za %d minuta
 
 [tr]
-ADMIN_KICK_1 = ADMIN: at %s
-ADMIN_KICK_2 = ADMIN %s: at %s
-IP_REMOVED = Ip "%s" ban listesinden silindi
-AUTHID_REMOVED = Authid "%s" ban listesinden silindi
-ADMIN_UNBAN_1 = ADMIN: ban kalkdi %s
-ADMIN_UNBAN_2 = ADMIN %s: ban kalkdi %s
-ADMIN_ADDBAN_1 = ADMIN: banla %s
-ADMIN_ADDBAN_2 = ADMIN %s: banla %s
-BANNED = banlandi
-REASON = Neden
-FOR_MIN = %s dakika ban
-PERM = suresiz
-CLIENT_BANNED = Oyuncu "%s" banlandi
-ADMIN_SLAY_1 = ADMIN: oldurdu %s
-ADMIN_SLAY_2 = ADMIN %s: oldurdu %s
-CLIENT_SLAYED = Oyuncu "%s" olduruldu
-ADMIN_SLAP_1 = ADMIN: %s tokatla verilen zarar %d
-ADMIN_SLAP_2 = ADMIN %s: %s tokatla verilen zarar %d
-CLIENT_SLAPED = Oyuncu "%s" tokatlandi %d zarar ile
-MAP_NOT_FOUND = Map ismi bulunamadi yada mevcud degildir
-ADMIN_MAP_1 = ADMIN: %s mapina degisdirdi
-ADMIN_MAP_2 = ADMIN %s: %s mapina degisdirdi
-NO_MORE_CVARS = Rcon'a daha cvar ekliyemiyorsunuz!
+ADMIN_KICK_1 = ADMIN: %s attı 
+ADMIN_KICK_2 = ADMIN %s: %s attı
+IP_REMOVED = Ip "%s" yasak listeden silindi
+AUTHID_REMOVED = Hüviyetli "%s" yasak listeden silindi
+ADMIN_UNBAN_1 = ADMIN: %s yasağını kaldırdı
+ADMIN_UNBAN_2 = ADMIN %s: %s yasağını kaldırdı
+ADMIN_ADDBAN_1 = ADMIN: %s yasakladı
+ADMIN_ADDBAN_2 = ADMIN %s: %s yasakladı
+BANNED = men'edildi
+REAON = Sebep
+FOR_MIN = %s dakîkalık
+PERM = müddetsiz
+CLIENT_BANNED = Oyuncu "%s" men'edildi
+ADMIN_SLAY_1 = ADMIN: %s öldürdü
+ADMIN_SLAY_2 = ADMIN %s: %s öldürdü
+CLIENT_SLAYED = Oyuncu "%s" öldürüldü
+ADMIN_SLAP_1 = ADMIN: %s tokatla %d zarar verdi
+ADMIN_SLAP_2 = ADMIN %s: %s tokatla %d zarar verdi
+CLIENT_SLAPED = Oyuncu "%s" %d zarar ile tokatlandı
+MAP_NOT_FOUND = Harita ismi bulunamadı yada mevcut değil
+ADMIN_MAP_1 = ADMIN: %s haritaya değiştirdi
+ADMIN_MAP_2 = ADMIN %s: %s haritaya değiştirdi
+NO_MORE_CVARS = Rcon'a daha cvar ekleyemiyorsunuz!
 UNKNOWN_CVAR = Bilinmeyen cvar: %s
 UNKNOWN_XVAR = Bilinmeyen xvar: %s
-CVAR_NO_ACC = O cvar'a giris hakkiniz yok
-XVAR_NO_ACC = O xvar'a giris hakkiniz yok
+CVAR_NO_ACC = O cvar'a erisiminiz yok
+CVAR_NO_ACC = O xvar'a erisiminiz yok
```

Comment:
You changed the key name as well. The second one should be `XVAR_NO_ACC`, not `CVAR_NO_ACC`.

Reply:
I did not see